### PR TITLE
WRP-3080: Update readdirp dependency

### PR DESCRIPTION
**Pull request by andreitirlalge**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
After updating "readdirp" module version to "^3.6.0" a build error occured.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
It was fixed by refactoring the readdirp api call following this example: https://github.com/paulmillr/readdirp/tree/3.0.0

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-3080

### Comments
Enact-DCO-1.0-Signed-off-by: Andrei Tirla andrei.tirla@lgepartner.com

### CODE REVIEW

File: scripts/bootstrap.js [hunk 1]
```diff
@@ -1,38 +1,31 @@
...
-				shell.cd(file.fullParentDir);
-				const result = shell.exec('npm install', {silent: true});
-				if (result.code !== 0) {
-					console.log(`${file.parentDir} failed to bootstrap: ${result.stdout}`);
-					error = true;
-				}
+			const directory = path.dirname(file.fullPath);
+			console.log(`Bootstrapping ${directory}`);
+			shell.cd(directory);
+			const result = shell.exec('npm install && npm run pack', {silent: true});
```
Comment (adrian-cocoara-lgp) [inline scripts/bootstrap.js:20]:
```suggestion
			const result = shell.exec('npm install', {silent: true});
```

File: scripts/bootstrap.js [hunk 2]
```diff
@@ -1,38 +1,31 @@
...
+
+			if (result.code !== 0) {
+				console.log(`${file.fullPath} failed to build: ${result.stdout}`);
```
Comment (adrian-cocoara-lgp) [inline scripts/bootstrap.js:23]:
```suggestion
				console.log(`${file.fullPath} failed to bootstrap: ${result.stdout}`);
```

File: scripts/build.js [hunk 3]
```diff
@@ -1,38 +1,31 @@
...
-				shell.cd(file.fullParentDir);
-				const result = shell.exec('npm install && npm run pack', {silent: true});
-				if (result.code !== 0) {
-					console.log(`${file.parentDir} failed to build: ${result.stdout}`);
-					error = true;
-				}
+			const directory = path.dirname(file.fullPath);
+			console.log(`Building ${directory}`);
+			shell.cd(directory);
+			const result = shell.exec('npm install', {silent: true});
```
Comment (adrian-cocoara-lgp) [inline scripts/build.js:20]:
```suggestion
			const result = shell.exec(''npm install && npm run pack', {silent: true});
```

File: scripts/build.js [hunk 4]
```diff
@@ -1,38 +1,31 @@
...
+
+			if (result.code !== 0) {
+				console.log(`${file.fullPath} failed to bootstrap: ${result.stdout}`);
```
Comment (adrian-cocoara-lgp) [inline scripts/build.js:23]:
```suggestion
				console.log(`${file.fullPath} failed to build: ${result.stdout}`);
```

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Review by MikyungKim — COMMENTED** [review]
The package-lock.json is included in this PR. Is this intended?

**Comment by andreitirlalge** [comment]
> The package-lock.json is included in this PR. Is this intended?

Hello, the package-lock,json push was not intended. I will remove it! 
Thank you for this notice.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**